src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int pick = new Random().nextInt(DifferentColors.values().length);
+        return DifferentColors.values()[pick].toString();
```

Comment:
[it’s better to use standard method of enum name() that will be returning always String representation of concrete enum constant.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)